### DISCUSSION TITLE
service modual

### DISCUSSION THREAD

**Discussion by FuryFire2004** [zer0Kerbal/Dreamer#39] [Ideas]
![image](https://user-images.githubusercontent.com/30536275/158817135-50cdc267-ded5-4e8e-a5f2-ee793d656cb7.png)
can we consider adding this

**Comment by FuryFire2004** [comment]
I could when I have time model this actually although how textures work is a bit beyond my current knowledge

**Comment by zer0Kerbal** [comment]
Yes, a service module would be great. Ultimately it is about the model (.mu) and texture (.png/tga/mbm/dds) - 

Think of a piece of candy. Textures are the wrapper. It is just that easy. (with nuances like emissive and the funny flip/rotate of converting to .dds and the whole DX1 DX5 - which compression)

**Reply by FuryFire2004** [reply]
yea but idk if i have time to or not

**Reply by FuryFire2004** [reply]
mabye for a future mod update? (after 1.12.3 update)

**Reply by zer0Kerbal** [reply]
this won't be the last update for Dreamer. Too much to do already in order to get it out the door possibly this weekend. I need to get it done (and Radial Heat Shields) so I can work on the next releases of MoarKerbals, SYD,OHS,KAMP,KPM,and so many other. all code. my 🪦 awaits.

**Comment by zer0Kerbal** [comment]
WhimChaser has two that might work. Same author.
